client/sysinfo/src/sysinfo.rs

```diff
@@ -504,7 +589,11 @@ pub fn benchmark_sr25519_verify(limit: ExecutionLimit) -> Throughput {
 /// Benchmarks the hardware and returns the results of those benchmarks.
 ///
 /// Optionally accepts a path to a `scratch_directory` to use to benchmark the disk.
-pub fn gather_hwbench(scratch_directory: Option<&Path>) -> HwBench {
+pub fn gather_hwbench(
```

Comment:
Please update the docs on what the new args mean and how the functionality is supposed to help users.